### DISCUSSION TITLE
master有个小问题问一下

### DISCUSSION THREAD

**Discussion by 281677160** [immortalwrt/immortalwrt#832] [Q&A]
请问大佬，master的 luci 跟 default-settings-chn 能设置成默认勾选吗？
我看了下include/target.mk又有啊，但是他们好像不生效
![QQ截图20221202164508](https://user-images.githubusercontent.com/66651997/205253462-b3130728-33ed-4f34-89e6-301cef8c3d05.png)
![QQ截图20221202164524](https://user-images.githubusercontent.com/66651997/205253475-8bba710b-6bae-496a-a9ff-b944dc84077b.png)

**Comment by 1715173329** [comment]
永远不会。master 仅作开发用途。
